views/partials/prs.hbs

```diff
@@ -46,5 +46,9 @@
             {{/each}}
         </div>
     {{/if}}
-{{/exists}}
 
+    <div class="br2 center mt5 overflow-hidden w-50-ns w-80-m w-90 mt2">
+        <button class="bg-orange white center mt2 h2 bn pointer br1 pv0 ph3 db-ns" onclick='saveUserPage()'>This is Me</button>
```

Comment:
Can we use an on click handler to do the save user paging, we have zepto included so should be able to do something like:

```js
button.on('click', function(e) {
    saveUserPage();
});
```

You may want to put an ID on the button.

Reply:
What about es6 arrow function feature?
```js
button.on('click', e => saveUserPage());
```